website/docs/migrating-from-docker/using-podman-mac-helper.md

```diff
@@ -29,17 +29,24 @@ The service redirects `/var/run/docker` to the fixed user-assigned UNIX socket l
 1. Install the `podman-mac-helper` tool.
    Run the command:
 
-    ```
-    $ brew install podman-mac-helper
+    ```sh
+    brew install podman-mac-helper
```

Comment:
@themr0c there is an error in the documentation as the helper is installed by podman, there is no formulae in brew to install the helper